src/style.css

```
@@ -1 +1,45 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
```

The body hasn't a padding
```suggestion
body {
  margin: 0;
}
```